internal/generate/namevaluesfilters/generators/servicefilters/main.go

```diff
@@ -29,10 +29,10 @@ var sliceServiceNames = []string{
 	"imagebuilder",
 	"licensemanager",
 	"neptune",
-	"opensearchservice",
```

Comment:
Looks like it was added to the generator's list but never actually generated. I removed it from the list to prevent the generator from generating code for it.